broadinstitute/orsp-pub
# BTRX - 790 - Fix legacy institutional resource

## Addresses
[BTRX-790](https://broadinstitute.atlassian.net/browse/BTRX-790): Institutional Sources not working on Legacy Sample/Data Cohorts
## Changes
- Added an extra condition for legacy projects where `institutionalSources` does not exist on the database.

## Testing
- Open a legacy consent group or a consent group that doesn't have any institutional Source stored in the data base, add a new element, the should allow to do this.
- Create a new consent group, its functionality shouldn't have changed

---
